packages/inter-protocol/test/snapshots/test-inter-cli.js.md

```diff
@@ -0,0 +1,111 @@
+# Snapshot report for `test/test-inter-cli.js`
+
+The actual snapshot is saved in `test-inter-cli.js.snap`.
```

Comment:
should the actual snapshot be checked in or  .gitignored?

Reply:
My read of the [ava snapshot testing docs](https://github.com/avajs/ava/blob/main/docs/04-snapshot-testing.md) is that you _must_ check in the `.snap` file, as that's what determines whether the test passes or fails, and you _should_ check in the `.md` file so that you can get a text representation of what changed.